File: ProyectoFinal4S/FrmMenu.cs
using System;
using System.Windows.Forms;

namespace ProyectoFinal4S.ProyectoFinal4S
{
    public partial class FrmMenu : Form
    {
        public FrmMenu()
        {
            InitializeComponent();
        }

        private void btnAPI_Click(object sender, EventArgs e)
        {
            Form form = new FrmAPI();
            form.Show();
            Hide();
        }

        private void btnDataSet_Click(object sender, EventArgs e)
        {
            Form form = new FrmDataSet();
            form.Show();
            Hide();
        }
    }
}

File: ProyectoFinal4S/FrmMenu.vb
Imports System
Imports System.Windows.Forms

Namespace ProyectoFinal4S
    Public Partial Class FrmMenu
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub btnAPI_Click(sender As Object, e As EventArgs)
            Dim form As Form = New FrmAPI()
            form.Show()
            Hide()
        End Sub

        Private Sub btnDataSet_Click(sender As Object, e As EventArgs)
            Dim form As Form = New FrmDataSet()
            form.Show()
            Hide()
        End Sub
    End Class
End Namespace
